napari/layers/surface/_tests/test_surface.py

```diff
@@ -311,7 +311,7 @@ def test_surface_normals():
     faces = np.array([[0, 1, 2], [3, 4, 5], [6, 7, 8]])
     values = np.array([1, 2, 3, 1, 2, 3, 1, 2, 3])
 
-    normals = dict(face=dict(visible=True, color='red'))
+    normals = {"face": {"visible": True, "color": 'red'}}
```

Comment:
Can you revert these unrelated changes @davidpross?

Reply:
Those are from the new ruff linter I think?

Reply:
Yes I recognise them, but why are they in this PR? Nothing else touches that file?

Reply:
Good question. I merged main to have tests pass and pre-commit fired.
Maybe a rule change in the linter? maybe @Czaki has an idea?

Reply:
I have merged the main branch as this change was already there.

Reply:
Thanks for figuring it out, I didn't recognize this change.